modules/aggregation-layers/bundle.js

```diff
@@ -8,5 +8,7 @@ const deck = _global.deck || {};
 if (!deck.LineLayer) {
   throw new Error('@deck.gl/layers is not found');
 }
+// Merge experimental exports
+Object.assign(deckGLLayers.experimental, deck.experimental);
```

Comment:
If the variable was called aggregationLayers I would have been less confused...

Didn't we drop experimental export in favor of underscores?

Reply:
We moved classes to underscore but kept utility functions in the `experimental` object.